dist/linux/ghostty_nautilus.py

```diff
@@ -20,7 +20,13 @@
 from os.path import isdir
 from gi import require_version
 from gi.repository import Nautilus, GObject, Gio, GLib
+import os
+import gettext
 
+DOMAIN = "com.mitchellh.ghostty"
+share_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I think this would be more readable with pathlib:

```python
from pathlib import path

locale_dir = Path(__file__).absolute().parent.parent.parent / "locale"
```

And does the Ghostty build system even guarantee that the locales always end up at this path, relative to the nautilus extension directory?

Reply:
@trag1c Oh, thanks 🙏 I learn a new thing every day 🙂